apps/web/modules/customization/ui/views/customization-view.tsx

```
@@ -0,0 +1,46 @@
+"use client";
+
+import { useQuery } from "convex/react";
+import { Loader2Icon } from "lucide-react";
+
+import { api } from "@workspace/backend/_generated/api";
+
+import { CustomizationForm } from "../components/customization-form";
+
+interface CustomizationViewProps {}
+
+export const CustomizationView = ({}: CustomizationViewProps) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)